src/Twilio/TwiML/FaxResponse.cs

```diff
@@ -25,17 +25,6 @@ public FaxResponse() : base("Response")
         {
         }
 
-        /// <summary>
-        /// Append a <Receive/> element as a child of this element
-        /// </summary>
-        /// <param name="receive"> A Receive instance. </param>
-        [System.Obsolete("This method is deprecated, use .Append() instead.")]
-        public FaxResponse Receive(Receive receive)
```

Comment:
Why did this get moved down?

Reply:
Because I refactored the creation of legacy methods in Yoyodyne to a function that executes after the renamed method is built.